docs/superpowers/specs/2026-04-13-zitadel-migration-design.md

```
@@ -0,0 +1,457 @@
+
+**`deploy/zitadel/zitadel-init-steps.yaml`:**
+Auto-creates on first boot:
+- Default OIDC application `raven-app` with:
+  - Redirect URIs: `https://app.ravencloak.org/callback`, `http://localhost:5173/callback`
+  - Post-logout URIs: `https://app.ravencloak.org`, `http://localhost:5173`
+  - Grant type: Authorization Code + PKCE
+  - Response type: Code
+- Machine user `raven-api` with key (for backend-to-Zitadel API calls if needed)
+
+**`deploy/zitadel/zitadel-secrets.yaml`:**
+```yaml
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)